### ISSUE TITLE
Do not encode and include content of large image files

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-server#11662]
This is related to https://github.com/mozilla/addons-code-manager/issues/856, which reports a problem when viewing a page that is for a large image file. The `JSON.parse` that is reported is an indication that the app was unable to successfully receive the API response.

We discussed this in Whistler and it was suggested that the API should not be encoding and sending file content that is this large.

The code for viewing images in code-manager will need to be updated to not rely on the file content to enable this change, and https://github.com/mozilla/addons-code-manager/issues/858 has been opened to track that.

**Comment by EnTeQuAk** [comment]
> We discussed this in Whistler and it was suggested that the API should not be encoding and sending file content that is this large.

Additionally, I'd like to not send any file-content for files that can be previewed by using the download-url, e.g images. Just to mention that, I think it's implicitly part of this issue anyway though.

**Comment by bobsilverberg** [comment]
@EnTeQuAk https://github.com/mozilla/addons-code-manager/issues/858 has landed on dev. We should probably wait for it to be verified before proceeding with this issue, but hopefully that will happen soon.

**Comment by EnTeQuAk** [comment]
This actually got resolved by https://github.com/mozilla/addons-server/issues/11782 - which I merged on Saturday and maybe that's because no one has seen issues with images in the meantime :see_no_evil: It may have introduced a needless performance regression as it's calculating the mimetype needlessly but that's a different issue.

I'm going to close this but keep it in the milestone to verify and have everyone being aware.

tldr; `content` should now **always** be an empty string for binary files like images and others. Only text-like files will be rendering `content`. SVGs for example count as images

**Comment by AlexandraMoga** [comment]
This has been verified as part of #11782. Also, https://github.com/mozilla/addons-code-manager/issues/856 can't be reproduced anymore.
I'm marking the issue as verified fixed.
